### PR TITLE
DOCS-10239-db-snowflake-sp-kt

### PR DESCRIPTION
**Pull request by kevintroller**
Updating Database Conenctor Stored Procedure documentation and Database Connector Troubleshooting page with new information regarding Snowflake database formatting the callable statement for stored procedures.

### CODE REVIEW

**Review by stefangretcan — DISMISSED** [review]

File: db/1.9/modules/ROOT/pages/database-connector-troubleshooting.adoc [hunk 1]
```diff
@@ -42,6 +42,23 @@ The following example shows how to use pooling profiles with the *Stored procedu
 	</flow>
 ----
 
+== Snowflake JDBC Driver Throws Exception for Stored Procedure
+
+Some drivers have their own specifications when calling stored procedures. When you set the callable statement with curly brackets `{` and `}` for the Snowflake JDBC driver, for example:
```
Comment (valkyrie69) [inline db/1.9/modules/ROOT/pages/database-connector-troubleshooting.adoc]: Running the application when you set the callable statement with curly brackets ({ and }), for example, `{call doubleMyInt(:myInt)}`, causes the Snowflake JDBC driver to throw the following exception:

File: db/1.9/modules/ROOT/pages/database-connector-troubleshooting.adoc [hunk 2]
```diff
@@ -42,6 +42,23 @@ The following example shows how to use pooling profiles with the *Stored procedu
...
+`{call doubleMyInt(:myInt)}`
+
+Running the application, causes the Snowflake JDBC driver to throw the exception:
+
+`net.snowflake.client.jdbc.SnowflakeSQLException: Unsupported feature 'call'`
+
+
+=== Solution
+
+For Snowflake JDBC driver set the callable statement without the curly brackets `{` and `}`:
```
Comment (valkyrie69) [inline db/1.9/modules/ROOT/pages/database-connector-troubleshooting.adoc]:
```suggestion
For Snowflake JDBC driver, set the callable statement without the curly brackets `{` and `}`, for example:
```

File: db/1.9/modules/ROOT/pages/database-connector-troubleshooting.adoc [hunk 3]
```diff
@@ -42,6 +42,23 @@ The following example shows how to use pooling profiles with the *Stored procedu
 	</flow>
 ----
 
+== Snowflake JDBC Driver Throws Exception for Stored Procedure
```
Comment (valkyrie69) [inline db/1.9/modules/ROOT/pages/database-connector-troubleshooting.adoc]:
```suggestion
== Snowflake JDBC Driver Throws an Exception for a Stored Procedure
```

File: db/1.9/modules/ROOT/pages/database-stored-procedure.adoc [hunk 4]
```diff
@@ -12,6 +12,11 @@ Database Connector supports the use of callable statements with the format:
 `{ call procedureName(:param1, :param2, ..., :paramN) }`
 
 Where `:paramN` matches the `Nth` parameter in the stored procedure declaration. Make sure that no whitespace is between the name of the stored procedure and the first parentheses.
+Even though all JDBC drivers must comply with the JDBC specification format, some drivers have their own specifications. Check the documentation of your JDBC driver for additional information on how to call the stored procedure. For example, for the Snowflake JDBC driver you set the callable statement without the curly brackets `{` and `}`:
```
Comment (valkyrie69) [inline db/1.9/modules/ROOT/pages/database-stored-procedure.adoc]:
```suggestion
Even though all JDBC drivers must comply with the JDBC specification format, some drivers have their own specifications. Check the documentation of your JDBC driver for additional information on how to call the stored procedure. For example, for the Snowflake JDBC driver you must set the callable statement without the curly brackets `{` and `}`:
```

**Review by valkyrie69 — DISMISSED** [review]

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**